src/Parlot/Fluent/ParseContext.Scoped.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Parlot.Fluent
+{
+    public abstract class ParseContext<TParseContext> : ParseContext
```

Comment:
`ScopedParseContext<TParseContext>`

Otherwise not obvious what it is and that it's only used by `ScopeParser`.

Reply:
indeed